server/src/__tests__/donatedItem.test.ts

```diff
@@ -0,0 +1,121 @@
+import request from 'supertest';
+import express from 'express';
+import donatedItemRoutes from '../routes/donatedItemRoutes';
+import mockPrismaClient from '../__mocks__/mockPrismaClient';
+
+const app = express();
+app.use(express.json());
+app.use('/donatedItem', donatedItemRoutes);
+
+describe('DonatedItem API Tests', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    // Test POST /donatedItem
+    it('validates Program name and Donor email before creating a donated item', async () => {
+        mockPrismaClient.program.findUnique.mockResolvedValue({ id: 1, name: 'Valid Program', description: 'Valid', startDate:'2024-10-04', aimAndCause:'recycle' });
+        const newDonor = {
```

Comment:
Can you move all the mock data across this file into seperate file/directory? It will facilitate us to reuse that data in many test files.